src/renderer/components/ft-shaka-video-player/ft-shaka-video-player.css

```
@@ -1599,6 +1599,11 @@
   z-index: 0;
 }
 
+.ftVideoPlayer .fullscreenActions:has(:deep(.fullscreenShareAction .iconDropdown)),
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":deep" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)